installing/installing_with_agent_based_installer/installing-using-iscsi.adoc

```diff
@@ -0,0 +1,59 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-using-iscsi"]
+= Preparing installation assets for iSCSI booting
+include::_attributes/common-attributes.adoc[]
+:context: installing-using-iscsi
+
+toc::[]
+
+You can boot an {product-title} cluster through iSCSI using an ISO image generated by the Agent-based Installer.
+The following procedures describe how to prepare the necessary installation resources for you to upload to an iSCSI target.
```

Comment:
We're reloading uploading to the target, which can be a separate iSCSI disk. So perhaps "resources to boot off of an iSCSI target"?